fix(bridge): Fix wrong time in sequence timeline (#7036)
keptn/keptn: bridge/README.md

To test your UI tests locally, use following commands:

> - `yarn cypress:open` (Linux, macOS), `yarn cypress:open:win32` (Windows) - Development of tests.<br/>This opens a browser where changes where you can run your tests and inspect them. This will be automatically updated on every code change made on the `*.spec.ts` files.

Reviewer: ```suggestion
- `yarn cypress:open` (Linux, macOS), `yarn cypress:open:win32` (Windows) - Used for the local development of tests.<br/>This opens a browser, where you can run your tests and inspect them. The tests will re-run automatically on every code change made on the `*.spec.ts` files.
```

> - or `yarn test:ui` (Linux, macOS), `yarn test:ui:win32` (Windows) - Headless browser mode also used in CI.<br/>This will just run the tests on a headless browser without the possibility to inspect them.

Reviewer: ```suggestion
- or `yarn test:ui` (Linux, macOS), `yarn test:ui:win32` (Windows) - This starts the headless browser mode that is also used in CI.<br/>This will just run the tests on a headless browser without the possibility to inspect them.
```
